src/Components/SectionBreakLine.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import '../Styles/SectionBreakLine.css'
+
+function SectionBreakLine() {
+  return (
+    <div>
+      <div className="sbl-container">
+        
+      </div>
+    </div>
+  )
```

Comment:
self-closing, and remove unnecessary div wrapper.
```suggestion
  return <div className="sbl-container" />;
```